<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>301</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <underline>true</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bienvenue au service de reservation de parking!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Connexion:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Creer un compte:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_UtilisateurConnexion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Courriel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MotDePasseConnexion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SeConnecter">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>210</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Se connecter</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>21</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_UtilisateurNouvCompte">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Courriel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MotDePasseNouvCompte">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_PlaqueNouvCompte">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Plaque d'immatriculation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CreerNouvCompte">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Creer le compte</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
